specs/AreBrowserAcceleratorKeysEnabled.md

```diff
@@ -0,0 +1,109 @@
+# AreBrowserAcceleratorKeysEnabled
+
+## Background
+In general, WebView2 tries to behave as much like the browser as possible.
```

Comment:
This spec talks about both WebView2 and CoreWebView2. It would help to be specific here.

```suggestion
In general, `CoreWebView2` tries to behave as much like the browser as possible.
```